app/components/status-filter/status-filter-directive.js

```diff
@@ -7,6 +7,7 @@ angular.module('statusFilter.directive', [
   .provider('StatusFilterOptions', function (moment) {
     var _statuses = [
       {label: 'Draft', key: 'status', value: 'draft'},
+      {label: 'Awaiting Review', key: 'status', value: 'Waiting for Editor'},
```

Comment:
edit: looks like they are both case sensitive and opposite... not sure how that happened